addons/web/static/src/scss/bs_mixins_overrides.scss

```diff
@@ -6,7 +6,8 @@
 // This variable must be defined here instead of bootstrap overridden files
 // otherwise we will have deprecation messages during assets generation
 $enable-deprecation-messages: false !default;
-
+$white: #FFF !default;
+$black: #000 !default;
```

Comment:
Should not this be on its own commit if this solves a bug? In stable maybe?

Reply:
That's create a bug only with my use of color-contrast function. Probably the necessary variables are not loaded at this point of time.

Reply:
I found the issue, my bad! The bundles wasn't loaded correctly when i've tested. I can remove these two lines.